src/node_contextify.cc

```diff
@@ -1108,6 +1109,26 @@ void ContextifyContext::CompileFunction(
   args.GetReturnValue().Set(fun);
 }
 
+void ContextifyContext::CreateCacheForFunction(
+    const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  // Argument 1: target function
+  Local<Function> function = args[0].As<Function>();
+
+  const std::unique_ptr<ScriptCompiler::CachedData> cache(
+      ScriptCompiler::CreateCodeCacheForFunction(function));
+
+  if (cache == nullptr) {
+    args.GetReturnValue().SetUndefined();
```

Comment:
nit: it's undefined by default so we can just set it if `cache != nullptr` and leave out an else block (I'd wish I could use the GitHub suggestion for this but there are too many lines and will result in tons of commits :()

Reply:
Haha, no worries. I got this.